src/tools/utils/format.ts

```diff
@@ -48,3 +48,11 @@ export const formatPullRequestList = (branches: Branch[]) => {
 		})
 		.join("\n");
 };
+
+export const formatTaskList = (tasks: Task[]) => {
+	return tasks
+		.map((task) => {
+			return `${task.complete ? "[X]" : "[ ]"} ${task.description}`;
```

Comment:
```suggestion
			return `- ${task.complete ? "[X]" : "[ ]"} ${task.description}`;
```

Reply:
Not required as part of this PR, but we might want to add `owner` as part of the task output.